src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -956,6 +957,8 @@ public function getSqlType($type, $limit = null)
                 return ['name' => 'bit', 'limit' => $limit ?: 64];
             case static::PHINX_TYPE_SMALL_INTEGER:
                 return ['name' => 'smallint', 'limit' => $limit ?: 6];
+            case static::PHINX_TYPE_TINY_INTEGER:
+                return ['name' => 'tinyint', 'limit' => $limit ?: 3];
```

Comment:
Isn't the default limit/length for tinyint in MySQL "4"?

Reply:
Is it? I thought 0...255 would be 3, but since it will fill one byte anyway you might be right.

Reply:
I did some research and the length here is the max display width, so given the negative char `-128` it seems to be 4 then.